p2p/permissions.go

```diff
@@ -24,7 +24,7 @@ func isNodePermissioned(nodename string, currentNode string, datadir string, dir
 		permissonedList = append(permissonedList, v.ID.String())
 	}
 
-	log.Debug("isNodePermissioned", "permisssionedList %v", permissonedList)
+	log.Debug("isNodePermissioned", "permisssionedList", permissonedList)
```

Comment:
permisssionedList typo

Reply:
This fixes the typo to use the new log format

Reply:
there are 3 s'es in permisssioned

Reply:
also, "permissonedList" is a typo